content/enterprise_influxdb/v1.8/about-the-project/release-notes-changelog.md

```diff
@@ -9,11 +9,24 @@ menu:
     parent: About the project
 ---
 
+## v1.8.5 [2020-04-19]
+
+The InfluxDB Enterprise 1.8.5 release builds on the InfluxDB OSS 1.8.5 release.
+For details on changes incorporated from the InfluxDB OSS release, see
+[InfluxDB OSS release notes](/influxdb/v1.8/about_the_project/releasenotes-changelog/#v185-2021-04-19).
+
+### Bug fixes
+
+- Resolve TSM backup "snapshot in progress" error.
+- Make meta queries respect QueryTimeout values.
+- Add labels to verbose output.
+- Always forward repairs to next node.
```

Comment:
meta and/or data node?

Reply:
As I understand what Mr. Norton meant, I would write: "...to the next data node, even if the current data node does not have to take any action itself for this repair"